backends-velox/src/test/scala/io/glutenproject/execution/VeloxTPCHSuite.scala

```diff
@@ -243,6 +243,11 @@ class VeloxTPCHV2BhjSuite extends VeloxTPCHSuite {
 }
 
 class VeloxPartitionedTableTPCHSuite extends VeloxTPCHSuite {
+
+  override protected def sparkConf: SparkConf = {
+    super.sparkConf.set("spark.memory.offHeap.size", "4g")
```

Comment:
this change is due to, velox dynamiac partition write requires more memory as it would create writer instance for each partition value.